ethcore/src/state/mod.rs

```diff
@@ -340,18 +340,18 @@ impl State {
 
 	/// Determine whether an account exists.
 	pub fn exists(&self, a: &Address) -> bool {
-		self.ensure_cached(a, RequireCache::None, |a| a.is_some())
+		self.ensure_cached(a, RequireCache::None, false, |a| a.is_some())
```

Comment:
could use a comment here explaining why this is consensus-critical/ensure that tests will catch it if changed.